packages/eslint-plugin-khan/src/rules/aphrodite-add-style-variable-name.ts

```diff
@@ -0,0 +1,73 @@
+import {TSESTree, ESLintUtils} from "@typescript-eslint/utils";
+
+import type {MyPluginDocs} from "../types";
+
+const createRule = ESLintUtils.RuleCreator<MyPluginDocs>(
+    (name) =>
+        `https://github.com/Khan/wonder-stuff/blob/main/packages/eslint-plugin-khan/docs/${name}.md`,
+);
+
+type Options = [];
+type MessageIds = "errorString";
+
+const message = `Variable name "{{ variableName }}" does not match tag name "{{ tagName }}". Variable name should be "{{ expectedName }}"`;
+
+export default createRule<Options, MessageIds>({
+    name: "aphrodite-add-style-variable-name",
+    meta: {
+        docs: {
+            description:
+                "Ensure variable names match the tag name passed to addStyle and follow the format: StyledTag (ie. StyledDiv, StyledImg)",
+            recommended: true,
+        },
+        messages: {
+            errorString: message,
+        },
+        schema: [],
+        type: "problem",
+    },
+    defaultOptions: [],
+    create(context) {
+        return {
+            VariableDeclarator(node: TSESTree.VariableDeclarator) {
+                // Check if addStyle is being called
+                if (
+                    node.init &&
+                    node.init.type === "CallExpression" &&
+                    node.init.callee.type === "Identifier" &&
+                    node.init.callee.name === "addStyle"
```

Comment:
style suggestion:  some of our other code compares `.type` fields to things like `AST_NODE_TYPES.Identifier`.  `AST_NODE_TYPES` can be imported from `"@typescript-eslint/utils"`.

Reply:
style suggestion: this could be one or early returns to avoid excessive indentation, e.g.
```ts
if (node?.init.type !== "CallExpression") {
  return;
}
if (node.init.callee.type !== "Identifier" || node.init.callee.name !== "addStyle) {
  return;
}
...
```

Reply:
Thanks for the feedback Kevin! I refactored the function to favour early returns. It is easier to read now 😄